docs/charts/parameter_estimate_comparisons_chart.ipynb

```diff
@@ -0,0 +1,221 @@
+       "</script>"
+      ],
+      "text/plain": [
+       "alt.Chart(...)"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from splink.duckdb.linker import DuckDBLinker\n",
+    "import splink.duckdb.comparison_library as cl\n",
+    "import splink.duckdb.comparison_template_library as ctl\n",
+    "import splink.duckdb.blocking_rule_library as brl\n",
+    "from splink.datasets import splink_datasets\n",
+    "import logging, sys\n",
+    "logging.disable(sys.maxsize)\n",
+    "\n",
+    "df = splink_datasets.fake_1000\n",
+    "\n",
+    "settings = {\n",
+    "    \"link_type\": \"dedupe_only\",\n",
+    "    \"blocking_rules_to_generate_predictions\": [\n",
+    "        brl.exact_match_rule(\"first_name\"),\n",
+    "        brl.exact_match_rule(\"surname\"),\n",
+    "    ],\n",
+    "    \"comparisons\": [\n",
+    "        ctl.name_comparison(\"first_name\"),\n",
+    "        ctl.name_comparison(\"surname\"),\n",
+    "        ctl.date_comparison(\"dob\", cast_strings_to_date=True),\n",
+    "        cl.exact_match(\"city\", term_frequency_adjustments=True),\n",
+    "        ctl.email_comparison(\"email\", include_username_fuzzy_level=False),\n",
+    "    ],\n",
+    "}\n",
+    "\n",
+    "linker = DuckDBLinker(df, settings)\n",
+    "linker.estimate_u_using_random_sampling(max_pairs=1e6)\n",
+    "\n",
+    "blocking_rule_for_training = brl.and_(\n",
+    "                            brl.exact_match_rule(\"first_name\"), \n",
+    "                            brl.exact_match_rule(\"surname\")\n",
+    "                            )\n",
+    "\n",
+    "linker.estimate_parameters_using_expectation_maximisation(blocking_rule_for_training)\n",
+    "\n",
+    "blocking_rule_for_training = brl.exact_match_rule(\"dob\")\n",
+    "linker.estimate_parameters_using_expectation_maximisation(blocking_rule_for_training)\n",
+    "\n",
+    "blocking_rule_for_training = brl.exact_match_rule(\"email\")\n",
+    "linker.estimate_parameters_using_expectation_maximisation(blocking_rule_for_training)\n",
+    "\n",
+    "\n",
+    "linker.parameter_estimate_comparisons_chart()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### What the chart shows\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### How to interpret the chart\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Actions to take as a result of the chart\n",
+    "\n"
+   ]
+  }
+ ],
```

Comment:
Do we want to include these if they're empty?

I know these specific workbooks are works in progress, but it feels and looks odd having so many blank cells.

I know this is only a temporary solution, so please do disagree if it makes your life easier/you think they should be left in.

Reply:
It is more there to ensure the structure is kept consistent across all the notebooks. I'm hoping to get all of the charts done this week so happy to leave as is for now.